doc/covid-response.Rmd

```diff
@@ -0,0 +1,106 @@
+
+## Assumptions and limitations
+
+1. We've restricted the time-line for consideration to be between March 1st 2020 and October 31st 2020.
+2. The choice of timeline is further explained in our Exploratory Data Analysis reports under the eda folder.
+3. We acknowledge that it is impossible to gather data regarding the true population of confirmed COVID-19 cases and the viral detection tests being conducted.
+4. Our sample sizes for both Canada and USA while calculating daily response ratios are limited by the data as present in our data set. 
+5. We acknowledge that the nature of our data set is temporal and that there will be autocorrelation within each country's response rate calculations.
+6. Our analysis seeks to establish a simple comparison with no attempts to infer causality and/or other overarching conclusions about either 
+
+## Methodology
+
+The team performed extensive EDA to wrangle the data and make it suitable for further analysis. Details of the EDA process are captured in our reports under the eda folder which can be found [here](https://github.com/UBC-MDS/covid-19-cases-vs-tests-analysis/tree/main/eda).
+
+Firstly, we calculated the daily response ratio for both Canada and USA for the time-line of interest using the daily new_tests and new_cases columns found in the data set. We analyzed the distribution of the daily response ratio for both the countries during the time-line of interest as shown in the figure below.
+
+![](../results/histogram_plot.png)
+Figure 1. Sample Distribution of Daily Response Ratio for USA and Canada between March 1st 2020 and October 31st 2020.
+
+We analyzed and confirmed that there is a difference in the variances of distributions of the daily response ratio data for both Canada and USA during the time-line of interest through a Bartlett's test. 
+
+The team decided to use permutation based hypothesis testing to answer our statistical question. We chose a 95% confidence level for our sample estimate i.e a significance threshold level of ${\alpha = 0.05}$.
+
+Our Null($H_0$) and Alternate($H_a$) Hypotheses are:
+
+$$ \text {$H_0$: The median daily response ratio of Canada and USA between March 1st 2020 and October 31st 2020 is equal} $$
+
+$$ \text {$H_a$: The median daily response ratio of Canada and USA between March 1st 2020 and October 31st 2020 is not equal} $$
+We generated a bootstrap distribution of bootstrap sample median response ratios for Canada and USA during the time-line of interest. We visualized our booststrap distributions as shown in the figure below.
+
+![](../results/median_simulation.png)
```

Comment:
Yes, I've it as an agenda item for tomorrow. Would also prefer to have some more aesthetic styling on the plots, such as smaller font size etc.

Reply:
Acknowledged. This should be automatically fixed when we change the code in script4 that generates the figure since this report references the plot in the results folder.